tala_connector.py

```diff
@@ -472,8 +477,12 @@ def _handle_create_scan(self, param):
         project_ids = param.get('project_ids', None)
 
         try:
-            project_ids_list = [int(x) for x in project_ids.split(",")]
-        except:
+            project_ids_list = []
+            for x in project_ids.split(","):
+                if int(x) < 0:
```

Comment:
Are we sure that project IDs should be > 0?

Reply:
Reverted back as I do not get a proper document about project id validation.